### ISSUE TITLE
Let's abandon the Swing designer

### ISSUE THREAD

**Issue by turnimator** [turnimator/forthide#7]
In my experience with Vaadin and Android, it is much easier to develop a complex application without the use of a visual designer.
For a quick and Dirty throwaway tool, the visual designer is great.
For the forth IDE, it's beginning to get in the way.

**Comment by motilio** [comment]
Should JavaFX be evaluated as an option? SWT? RCP?

**Comment by turnimator** [comment]
Nah. Swing is great, easy to work with and unproblematic in every way. It's just the designer tool I don't like. Let's create all the widgets programmatically, so we have full control over the layout.

**Comment by motilio** [comment]
Main Panel migrated to a pure Java class
